compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/KnownTargets.cpp

```diff
@@ -232,11 +232,44 @@ const WgpDetails *getCDNA1WgpDetails() {
   return &cdna1Wgp;
 }
 
+const WgpDetails *getRDNA4WgpDetails() {
+  // XXX: Actively incorrect, semantics change
```

Comment:
```suggestion
  // FIXME(@krzysz00): Actively incorrect, semantics changed.
```

Reply:
Actually I am not sure to parse this sentence. Are you saying that the semantics are still in flux? Maybe expand those words into a complete sentence.

Reply:
... Good news, that's a stray comment!

Reply:
(What happened is that I initially copied the RDNA3 target and left in a note saying that those weren't right so I wouldn't accidentally merge that. I fixed the problem but left the note )